index.html

```diff
@@ -3354,22 +3351,6 @@ <h2>
           consent.
         </p>
       </section>
-      <section>
-        <h2>
-          Exposing available payment methods
-        </h2>
-        <p>
-          Developers might try to call the payment request API repeatedly with
```

Comment:
Personally I find consolidating privacy considerations into a single section to be a good idea. The difference would be that you'd have a pointer to the parts of the algorithms that help preserve privacy, instead of adding MAY stuff there. So I would change this to include all the same helpful overview of the problem, then point to the mitigations we've put in place in show() and canMakePayment().